configs/fast_rcnn/README.md

```diff
@@ -15,49 +15,85 @@ This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN
 ## Introduction
 
 Before training the Fast R-CNN, users should first train an [RPN](../rpn/README.md), and use the RPN to extract the region proposals.
-
-- Firstly, extract the region proposals of the val set by this command as below:
-
-```bash
-./tools/dist_test.sh \
-    configs/rpn_r50_fpn_1x_coco.py \
-    checkpoints/rpn_r50_fpn_1x_coco_20200218-5525fa2e.pth \
-    8 \
-    --out proposals/rpn_r50_fpn_1x_val2017.pkl
-```
-
-- Then, change the `ann_file` and `img_prefix` of `data.test` in the RPN config to train set as below:
-
-```python
-data = dict(
-    test=dict(
-        ann_file='data/coco/annotations/instances_train2017.json',
-        img_prefix='data/coco/train2017/'))
-```
-
-- Extract the region proposals of the train set by this command as below:
-
-```bash
-./tools/dist_test.sh \
-    configs/rpn_r50_fpn_1x_coco.py \
-    checkpoints/rpn_r50_fpn_1x_coco_20200218-5525fa2e.pth \
-    8 \
-    --out proposals/rpn_r50_fpn_1x_train2017.pkl
-```
-
-- Modify the path of `proposal_file` in Fast R-CNN config as below:
-
-```python
-data = dict(
-    train=dict(
-        proposal_file='proposals/rpn_r50_fpn_1x_train2017.pkl'),
-    val=dict(
-        proposal_file='proposals/rpn_r50_fpn_1x_val2017.pkl'),
-    test=dict(
-        proposal_file='proposals/rpn_r50_fpn_1x_val2017.pkl'))
-```
-
-Finally, users can start training the Fast R-CNN.
+The region proposals result can be obtained by setting `DumpProposals` pseudo metric. The dumped results is a `dict(file_name: pred_instance)`.
+The `pred_instance` is a `InstanceData` with the ranked bboxes and scores calculate from RPN. We provide example of dumping proposals in [RPN config](../rpn/rpn_r50_fpn_1x_coco.py).
+
+- Firstly, it should be obtained the region proposals in both training and validation (or testing) set.
+  change the type of `test_evaluator` to `DumpProposals` in the RPN config to get the region proposals as below:
+
+  The config of get training image region proposals can be set as below:
+
+  ```python
+  # For training set
+  test_dataloader = dict(
+      dataset=dict(
+          ann_file='data/coco/annotations/instances_train2017.json',
+          data_prefix=dict(img='val2017/')))
+  test_evaluator = dict(
```

Comment:
```suggestion
  train_dataloader = dict(
      dataset=dict(
          ann_file='data/coco/annotations/instances_train2017.json',
          data_prefix=dict(img='val2017/')))
  train_evaluator = dict(
```

Reply:
should be test_evaluator, so that users can using tools/test.py to dump proposals.